addons/binding/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/dto/BridgeApiLockStateDto.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.nuki.dto;
+
+/**
+ * The {@link BridgeApiLockStateDto} class defines the Data Transfer Object (POJO) for the Nuki Bridge API /lockState
+ * endpoint.
+ *
+ * @author Markus Katter - Initial contribution
+ */
+public class BridgeApiLockStateDto {
+
+    private int state;
```

Comment:
What does this int mean, what is the valid range, can it be some other datatype like an enum or a boolean?